The Service {!Log_Error_Message__c.Sub_Type__c} ({!Log_Error_Message__c.Type__c}) has been failed with the following error : 
"{!Log_Error_Message__c.Error_message__c}"

Log Id: {!Log_Error_Message__c.Id}